bokehjs/test/framework.ts

```diff
@@ -258,7 +258,7 @@ async function _run_test(suites: Suite[], test: Test): Promise<PartialResult> {
       const {width, height} = test.view.layout.bbox
       const rect = test.el!.getBoundingClientRect()
       if (width > rect.width || height > rect.height)
-        throw new Error(`viewport size exceeded [${width}, ${height}] > [${rect.width}, ${rect.height}]`)
+        throw new Error(`${test.description}: viewport size exceeded [${width}, ${height}] > [${rect.width}, ${rect.height}]`)
```

Comment:
Why this change? This error should be printed directly below a nicely formatted test description. If that not the case, then there's an async issue somewhere in the code.